src/View/Widget/RadioWidget.php

```diff
@@ -172,6 +172,7 @@ protected function _renderInput($val, $text, $data, $context)
         }
         if (isset($data['val']) && (string)$data['val'] === (string)$radio['value']) {
             $radio['checked'] = true;
+            $radio['templateVars']['activeClass'] = "active";
```

Comment:
Any chance a test could be added to ensure this templateVar is not lost in future refactorings?